examples/components/Radar.vue

```diff
@@ -1,5 +1,18 @@
+/**
+  Radar is usually used to describe where we are looking at in 360 panorama viewers or VR projects,
+  inspired by krpano's radar plugin https://krpano.com/tours/bkeller/index.html
+  angle: field of view
+  direction: center angle of the view
+*/
+
```

Comment:
If I understand correctly, you don't need to drag&move you component along x and y positions, do you?
Just to be sure because actual demo allow drag&move and this may impact how you'll implement the rotation.

Reply:
I need to drag and move by dragging itself, and should be rotated by dragging its handler